reportportal_client/helpers.py

```diff
@@ -44,3 +46,31 @@ def gen_attributes(rp_attributes):
         logger.debug('Failed to process "{0}" attribute, attribute value'
                      ' should not be empty.'.format(rp_attr))
     return attrs
+
+
+def get_launch_sys_attrs():
+    """Generate attributes for the launch containing system information.
+
+    :return: dict {'os': 'Windows',
+                   'cpu': 'AMD',
+                   'machine': 'Windows10_pc'}
+    """
+    return {
+        'os': system(),
+        'cpu': processor() or 'unknown',
+        'machine': machine(),
+        'system': True  # This one is the flag for RP to hide these attributes
+    }
+
+
+def get_package_version(package_name):
+    """Get version of the given package.
+
+    :param package_name: Name of the package
+    :return:             Version of the package
+    """
+    try:
+        agent_version = get_distribution(package_name).version
```

Comment:
If we allow a lookup of any package name, then it is not necessarily an agent_version but rather a package_version?

Reply:
Agree.